README.md

```diff
@@ -0,0 +1,156 @@
+# To Do list:Version 1.4<!---Title of Project-->
+<a name="readme-top"></a>
+
+
+## React capstone project - Metrics webapp <!---Title of Project-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp<a name="about-project"></a> <!---Project title-->
+This React capstone project is about building a mobile web application to check a list of metrics that We've created making use of React and Redux. This website list the genres of popular shows and allows to see the shows descriptions.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+ React, Redux and CSS, and linter configuration setup.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo](moviesgenre.netlify.app)
```

Comment:
The movie link leads a broken link as shown below, this could be as a result of not giving the actual full reference. Kindly ensure that you add the correct and full reference before submiting for another review. :pray: 

![Screenshot from 2023-04-22 20-15-21](https://user-images.githubusercontent.com/65233860/233804637-00c41663-eca4-43a9-a39b-8dbb40835f8e.png)